install/helm-single/templates/backend.deployment.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: backend
+  name: backend-deployment
+  namespace: hydro-namespace
+spec:
+  replicas: {{.Values.Backend.Replicas}}
+  revisionHistoryLimit: 1
+  strategy:
+    type: RollingUpdate
+  selector:
+    matchLabels:
+      app: backend
+  template:
+    metadata:
+      labels:
+        app: backend
+    spec:
+      containers:
+        - name: oj-backend
+          image: {{.Values.Backend.Image}}
+          imagePullPolicy: IfNotPresent
+          volumeMounts:
+            - mountPath: /data/file
+              name: file-volume
+            - mountPath: /root/.hydro
+              name: backend-volume
+            - mountPath: /root/.hydro-config-map
+              name: config-json-volume
+          ports:
+            - name: server-port
+              containerPort: 80
+          command: ["/bin/sh", "-c"]
+          args:
+            - |
+              cp /root/.hydro-config-map/addon.json /root/.hydro/ &&
+              cp /root/.hydro-config-map/config.json /root/.hydro/ &&
+              pm2-runtime start hydrooj
```

Comment:
If you are using a container to manage all the processes, it's recommended to drop pm2 and simply start with `hydrooj`.

NOTE: pm2 supports forking multiple app processes and allow them listening on the same port to provide load balancing functionality, is there an alternative option for k8s?